Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -7741,6 +7770,18 @@ void WebPageProxy::didReceiveEvent(uint32_t opaqueType, bool handled)
         auto event = m_mouseEventQueue.takeFirst();
         MESSAGE_CHECK(m_process, type == event.type());
 
+#if ENABLE(CONTEXT_MENU_EVENT)
+        if (event.button() == WebMouseEvent::RightButton) {
```

Comment:
Does this work with control-click on iPad?

Reply:
AFAICT yes it does